src/nrnpython/rxd_extracellular.cpp

```diff
@@ -1175,9 +1175,11 @@ static void ecs_dg_adi_x(ECS_Grid_node* g, const double dt, const int y, const i
     }
     for(x=1; x<g->size_x-1; x++)
     {
+#ifndef __PGI
         __builtin_prefetch(&(g->states[IDX(x+PREFETCH,y,z)]), 0, 1);
         __builtin_prefetch(&(g->states[IDX(x+PREFETCH,yp,z)]), 0, 0);
-        __builtin_prefetch(&(g->states[IDX(x+PREFETCH,ym,z)]), 0, 0);
+       	__builtin_prefetch(&(g->states[IDX(x+PREFETCH,ym,z)]), 0, 0);
```

Comment:
curious how the indent changed.

Reply:
It was just a mistake. Now it's fixed :+1: